packages/editor-ui/src/Interface.ts

```diff
@@ -1758,6 +1758,7 @@ export declare namespace Cloud {
 		username: string;
 		email: string;
 		hasEarlyAccess?: boolean;
+		role?: string;
```

Comment:
This might be easy to confuse with the new RBAC roles. Where is this data defined? We can either rename the property or add a more strict type for it, e.g. `CloudRole` or `RoleAtCompany`

```ts
type CloudRole = string;

export type UserAccount = {
  ...
  role?: CloudRole
}
```

Reply:
Makes sense, but this interface is already in the `Cloud` namespace. Does this make it clearer?